### DISCUSSION TITLE
Github: Use more labels to categorize issues

### DISCUSSION THREAD

**Discussion by NotSoImportant** [osmandapp/OsmAnd#11557] [Ideas]
Like this people can get a better overview about related issues inside of the current 1.9k issues.
I can offer my help to categorize some of them.

Incomplete list of suggestions, please add/refine:
- navigation
- downloads/live
- ux
- favorites
- my places
- configure map/screen
- profiles
- style
- osm entities
- wiki* (wikipedia, wikimedia, wikidata, wikivoyage)
- plugin: online maps
- plugin: ...
- type: bug
- type: feature
- type: routing

**Comment by vshcherb** [comment]
Used to do it in the past, but don't see much sense actually. Probably it helps to find duplicates or related issues but I feel general search works anyway better. Anyway it's worth a discussion and not an issue. I think most of the issues beyond 1-2 release plan should be converted to dicussions

**Reply by NotSoImportant** [reply]
thanks for this guidance. I was actually wondering what is the judgement to move it between issue and discussion :-)

**Comment by NotSoImportant** [comment]
To me it's less about finding duplicates, but to find related topics and ideas, so that there can be a harmonized solution.

**Reply by vshcherb** [reply]
Discussion is a new stream which I think to use mostly for discussions and not yet planned issues to fix. Probably with combinations of issues, I would like to clean up issues, so description either should be crystally clear how & what should be done or should be maintained by OsmAnd Team followed up by more broad discussion. 


So, in some sense issues will be something that will definitely be fixed at one moment but discussion / suggestion / idea is something that could not be fixed or hard /impossible to implement in first interpretation.

**Reply by NotSoImportant** [reply]
Yes this sounds lika a good idea
